pages/Types.html

```diff
@@ -411,6 +412,9 @@ <h3 id="Array.3CType.3E_Notation"> Array&lt;Type&gt; Notation </h3>
 </pre>
 <p>This indicates that the method doesn't only expect an array as the argument, but also specifies the expected type. The notation is borrowed from Java 5's generics notation (or C++ templates).
 </p>
+<h2 id="ArrayLikeObjects">Array-Like Objects</h2>
+<p>Either a true JavaScript Array or a JavaScript Object that contains a numeric <code>length</code> property. This latter case includes array-like objects commonly encountered in web-based code such as the <code>arguments</code> object and the <code>NodeList</code> object returned by many DOM methods.</p>
```

Comment:
We can't iterate on non-numeric properties, whereas `merge` docs says

> Merge the contents of two arrays together into the first array.

if there would be `array-like` there, it should, according to this definition of `array-like` do this:
```js
var a = { test: 1 };
a.length = 1;

console.log( $.merge( [], a ).test ); // 1
```

since "test" property is also "content" of the array-like object.